addons/modules/stringtable.xml

```diff
@@ -315,125 +336,143 @@
             <Chinese>移動</Chinese>
             <Chinesesimp>移动</Chinesesimp>
             <Russian>Выдвинуться</Russian>
+            <German>Bewegen</German>
         </Key>
         <Key ID="STR_CBA_Modules_SAD">
             <English>Seek and Destroy</English>
             <Japanese>索敵と破壊</Japanese>
             <Chinese>搜索並摧毀</Chinese>
             <Chinesesimp>搜索并摧毁</Chinesesimp>
             <Russian>Найти и Уничтожить</Russian>
+            <German>Suchen und Zerstören</German>
         </Key>
         <Key ID="STR_CBA_Modules_Loiter">
             <English>Loiter</English>
             <Japanese>周回</Japanese>
             <Chinese>盤旋</Chinese>
             <Chinesesimp>盘旋</Chinesesimp>
             <Russian>Слоняться без дела</Russian>
+            <German>Herumlungern</German>
```

Comment:
I would not translate this literally:
https://community.bistudio.com/wiki/Waypoint_types#Loiter

Reply:
Wie heißt denn der Loiter-Wegpunkt auf Deutsch?